### PR TITLE
Improve OSS metadata, refactor

### PR DESCRIPTION
**Pull request by chshersh**

### CODE REVIEW

File: cabal.project [hunk 1]
```diff
@@ -1,3 +1 @@
 packages: summoner-cli/ summoner-tui/
-
-tests: true
```
[the remark this answers is not in the document]
Comment (chshersh) [reply cabal.project]: No, they didn't. I actually deleted these lines accidentally, sorry for that...

File: summoner-cli/src/Summoner.hs [hunk 2]
```diff
@@ -6,12 +14,15 @@ import Summoner.Ansi as Summoner
...
 import Summoner.Config as Summoner
 import Summoner.CustomPrelude as Summoner
+import Summoner.Decision as Summoner
 import Summoner.Default as Summoner
 import Summoner.GhcVer as Summoner
 import Summoner.License as Summoner
 import Summoner.Project as Summoner
 import Summoner.Question as Summoner
 import Summoner.Settings as Summoner
+-- QUESION: Conflicting imports with Tree (File); should we rename something?
```
[the remark this answers is not in the document]
Comment (chshersh) [reply summoner-cli/src/Summoner.hs]: Yeah, I had the same idea. Maybe we can call it `Copy`? We already removed `Link` constructor, so it's kinda okay to rename one more constructor... Also, I don't think that `File` is used that often, so it should be okay to rename it.

File: summoner-cli/src/Summoner/Template.hs [hunk 3]
```diff
@@ -1,4 +1,10 @@
--- | This module contains functions for stack template creation.
+{- |
+Copyright: (c) 2017-2019 Kowainik
+SPDX-License-Identifier: MPL-2.0
+Maintainer: Kowainik <xrom.xkov@gmail.com>
+
+This module contains functions for stack template creation.
```
[the remark this answers is not in the document]
Comment (chshersh) [reply summoner-cli/src/Summoner/Template.hs]: Yes, it seems so :slightly_smiling_face: This line is so 2017....

File: cabal.project (hunk 1 above)
[the remark this answers is not in the document]
Comment (vrom911) [reply cabal.project]: No worries!

File: summoner-cli/src/Summoner.hs (hunk 2 above)
[the remark this answers is not in the document]
Comment (vrom911) [reply summoner-cli/src/Summoner.hs]: or Local?

[the remark this answers is not in the document]
Comment (chshersh) [reply summoner-cli/src/Summoner.hs]: I went for `Local`, I like it a lot :+1:

File: summoner-cli/CHANGELOG.md [hunk 4]
```diff
@@ -73,6 +73,7 @@ The changelog is available [on GitHub][2].
   want.
 * [#327](https://github.com/kowainik/summoner/issues/327):
   Better AppVeyor CI configuration for both `cabal` and `stack`.
+* Rename `file` confing file of `source` to `local`.
```
Comment (vrom911) [inline summoner-cli/CHANGELOG.md]: Should we add `Breaking change` here?

**Pull request merged**